### ISSUE TITLE
XSS vulnerability

### ISSUE THREAD

**Issue by dhruvgognaa** [linkedin/dustjs#740]
While playing around with the filters on http://www.dustjs.com/docs/syntax/ under "{reference}" section, I noticed that if I change the value of "markup" to "<img src=x onerror=alert(125)>", the alert box pops up, and none of the filters (s, j, h) resolve the issue (Sorry it even came to hit and trial to see if I was missing anything on my end). Not sure if the team is aware of the issue, but wanted to put out there

**Comment by sethkinast** [comment]
@smfoote can you peek at this?

**Comment by dhruvgognaa** [comment]
Wellll whad'ya know, "markup" is supposed to be <img src=x onerror=alert(123)>

**Comment by smfoote** [comment]
As far as I can tell, there is an error in how the demo web component is escaping HTML in the JSON. I don't believe this is an XSS hole in Dust itself. I'll keep researching.

**Comment by sethkinast** [comment]
Yeah I think it's definitely just the component, that was why I thought you
might know best how to handle it. It's just a self xss so not critical, no
worries

**Comment by smfoote** [comment]
It's a weird one, and now I'm curious.
